Docs/source/running_cpp/parameters.rst

```diff
@@ -400,6 +400,16 @@ Particle initialization
       ``<species_name>.single_particle_vel`` (`3 doubles`, particle 3D normalized momentum, i.e. :math:`\gamma \beta`)
       ``<species_name>.single_particle_weight`` ( `double`, macroparticle weight, i.e. number of physical particles it represents)
 
+    * ``MultipleParticles``: Inject multiple macroparticles.
```

Comment:
Just one minor comment: here the injection styles are called `SingleParticle` and `MultipleParticles`, however in `PlasmaInjector.cpp` we write `part_pos_s == "singleparticle"` and  `part_pos_s == "multipleparticles"`. I guess that the parser is **case-insensitive** for these directives (although it is **case-sensitive** for user-defined constants), but this is never mentioned in `https://warpx.readthedocs.io/en/latest/running_cpp/parameters.html`. Maybe we should add somewhere in that page a paragraph to explain what is case-insensitive and what is case-sensitive...

Reply:
@lucafedeli88 You raise a good point. That should be done is a separate PR. As part of that a check should be done to make sure that in fact WarpX is consistent in how it handles case sensitivity.

Reply:
I agree with you